java/core/src/main/java/com/google/protobuf/ByteString.java

```diff
@@ -279,8 +316,13 @@ public int compare(ByteString former, ByteString latter) {
       };
 
   /**
+<<<<<<< HEAD
    * Returns a {@link Comparator} which compares {@link ByteString}-s lexicographically
    * as sequences of unsigned byte values between 0 and 255, inclusive.
+=======
+   * Returns a {@link Comparator} which compares {@link ByteString}-s lexicographically as sequences
```

Comment:
I think the resolution this conflict should be something like this:
```
   * Returns a {@link Comparator} which compares {@link ByteString}-s lexicographically as sequences
   * of unsigned byte values between 0 and 255, inclusive.
```

Reply:
Fixed.